### ISSUE TITLE
[Feature Request] Add livestream button

### ISSUE THREAD

**Issue by toby63** [adamreichold/QMediathekView#21]
While you can of course search for "livestream" in the Topic search field, it would be very convenient to have a button to show all available livestreams.

**Comment by probonopd** [comment]
Good idea! Didn't know that one can search for "livestream" in the Topic search field and it works.
